sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md

```diff
@@ -3,6 +3,11 @@
 ## Current version
 ###  Feature Additions
 
+## Version 4.0.1
```

Comment:
This version i snot needed, 4.0.0 has not been released yet.
You don't need these change lines at all. Instead please update the type names in 4.0.0 and also update the release notes section in the csproj file, since it will be populated to nuget.

Reply:
updated. thanks